fodmap-project/app/globals.css

```diff
@@ -0,0 +1,57 @@
+html, body {
+  margin: 0;
+  padding: 0;
+}
+
+
+
+/* Common graph styling */
+
+
```

Comment:
if possible all code below should be encapsulated with components

Reply:
This code will be moved to the relevant page (i.e. the stats page) once I have created the components for the stats page. I have left it there as a reminder to myself but will remove it to avoid any issues.